src/lib.rs

```diff
@@ -39,6 +40,34 @@ pub struct Fernet {
 #[derive(Debug, PartialEq, Eq)]
 pub struct DecryptionError;
 
+pub struct MultiFernet {
+    fernets: Vec<Fernet>,
+}
+
+impl MultiFernet {
+    pub fn new(keys: Vec<Fernet>) -> MultiFernet {
+        assert!(keys.len() > 0);
+        MultiFernet {
+            fernets: keys,
+        }
+    }
+
+    pub fn encrypt(&self, data: &[u8]) -> String {
+        self.fernets.first().map(|f| f.encrypt(data)).unwrap()
```

Comment:
Can this simplify to `self.fernets[0].encrypt(data)`?

Reply:
Yup, fixed.